test/baggage/test_baggage.py

```diff
@@ -0,0 +1,181 @@
+import unittest
+import urllib.parse
+
+from baggage import Baggage, BaggageEntry
+
+
+class BaggageTest(unittest.TestCase):
```

Comment:
This is a good set of tests, thanks for adding them. Are you planning to add any limits related tests, or is the plan to add them after we finish the discussion around min/max limits?

Reply:
Yes the plan is to add limits, and to add a client-server test harness